sns_ik_lib/src/sns_position_ik.cpp

```diff
@@ -30,7 +30,7 @@ SNSPositionIK::SNSPositionIK(KDL::Chain chain, std::shared_ptr<SNSVelocityIK> ve
     m_jacobianSolver(chain),
     m_linearMaxStepSize(0.2),
     m_angularMaxStepSize(0.2),
-    m_maxIterations(150),
+    m_maxIterations(100),
```

Comment:
The solver will now fail faster. I tested different max iterations, and found that anything beyond 100 didn't really increase the success rate but increased the average run time.